spec/services/container_topology_service_spec.rb

```diff
@@ -40,8 +43,9 @@
     end
 
     it "topology contains the expected structure and content" do
-      ems_kube = FactoryGirl.create(:ems_kubernetes)
       ems_rhev = FactoryGirl.create(:ems_redhat, :name => "ems_rhev")
+      ems_kube = FactoryGirl.create(:ems_kubernetes, :name => "ems_kube", :id => 150)
```

Comment:
@abonas I think your issue might be here in setting the id.  You should not do that in specs for 2 reasons.

1. There is a possibility of id collision with the auto-sequencing
2. The specs should be region aware.  Specs on Travis are run with an id REGION of 1, thus, adding this region of 150 may not be recognized in certain code paths.  I also see some ids down below.  Please remove them all.  If you need ids for verification, keep a reference to the object you are creating on this line.